tests/unit/test_graph.cpp

```diff
@@ -395,16 +395,11 @@ TEST_F(GraphTest, RemoveNodes) {
 	uint edge_count = array_len(edges);
 	ASSERT_EQ(edge_count, 2);
 
-	// Delete the edges on the first node manually, as Graph_DeleteNode expects
-	// it to be detached.
-	// Both 0 -> 1 edge and 1 -> 0 edge should be removed.
-	for(uint i = 0; i < edge_count; i ++) Graph_DeleteEdge(g, &edges[i]);
+	Graph_GetNode(g, 0, &node);
+	Graph_BulkDelete(g, &node, 1, edges, edge_count, NULL, NULL);
```

Comment:
This test function `RemoveNodes` doesn't release the lock `Graph_AcquireWriteLock`